user_manual/groupware/mail.rst

```diff
@@ -19,6 +19,8 @@ Switch layout
 
 .. figure:: images/mail_layout.png
 
+When looking through your mailbox, if you have AI features enabled, you will see a short AI generated summary of your emails as a preview.
```

Comment:
Let's give this it's one section. It's a bit out of place here.

Change the phrazing to the style of the suggested replies. As in, document it like the feature exists, but add a note that admins have to set it up.

Reply:
Let's also utilize versionadded for this